rpm/containers-common.spec

```diff
@@ -0,0 +1,206 @@
+Suggests: fuse-overlayfs
+Source0: %{git0}/archive/v%{version_no_tilde}.tar.gz
+Source1: %{github_containers}/image/%{image_branch}/docs/containers-auth.json.5.md
+Source2: %{github_containers}/image/%{image_branch}/docs/containers-certs.d.5.md
+Source3: %{github_containers}/image/%{image_branch}/docs/containers-policy.json.5.md
+Source4: %{github_containers}/image/%{image_branch}/docs/containers-registries.conf.5.md
+Source5: %{github_containers}/image/%{image_branch}/docs/containers-registries.conf.d.5.md
+Source6: %{github_containers}/image/%{image_branch}/docs/containers-registries.d.5.md
+Source7: %{github_containers}/image/%{image_branch}/docs/containers-signature.5.md
+Source8: %{github_containers}/image/%{image_branch}/docs/containers-transports.5.md
+Source9: %{github_containers}/image/%{image_branch}/registries.conf
+Source10: %{github_containers}/shortnames/%{shortnames_branch}/shortnames.conf
+Source11: %{github_containers}/image/%{image_branch}/default.yaml
+Source12: %{github_containers}/image/%{image_branch}/default-policy.json
+Source13: %{github_containers}/storage/%{storage_branch}/docs/containers-storage.conf.5.md
+Source14: %{github_containers}/storage/%{storage_branch}/storage.conf
+# Fetch RPM-GPG-KEY-redhat-release from the authoritative source instead of storing
+# a copy in repo or dist-git. Depending on distribution-gpg-keys rpm is also
+# not an option because that package doesn't exist on CentOS Stream.
+Source15: https://access.redhat.com/security/data/fd431d51.txt
+
+%description
+This package contains common configuration files and documentation for container
+tools ecosystem, such as Podman, Buildah and Skopeo.
+
+It is required because the most of configuration files and docs come from projects
+which are vendored into Podman, Buildah, Skopeo, etc. but they are not packaged
+separately.
+
+%package extra
+Summary: Extra dependencies for Podman and Buildah
+Requires: %{name} = %{epoch}:%{version}-%{release}
+Requires: container-network-stack
+Requires: oci-runtime
+Conflicts: podman < 5:5.0.0~rc4-1
+Recommends: composefs
+Recommends: crun
+Requires: (crun if fedora-release-identity-server)
+Requires: netavark >= 1.10.3-1
+Suggests: slirp4netns
+Requires: passt
+Requires: iptables
+Requires: nftables
+Recommends: qemu-user-static
+Requires: (qemu-user-static-aarch64 if fedora-release-identity-server)
+Requires: (qemu-user-static-arm if fedora-release-identity-server)
+Requires: (qemu-user-static-x86 if fedora-release-identity-server)
+
+%description extra
+This subpackage will handle dependencies common to Podman and Buildah which are
+not required by Skopeo.
+
+%prep
+%autosetup -Sgit %{name}-%{version_no_tilde}
+cp %{SOURCE1} docs/.
+cp %{SOURCE2} docs/.
+cp %{SOURCE3} docs/.
+cp %{SOURCE4} docs/.
+cp %{SOURCE5} docs/.
+cp %{SOURCE6} docs/.
+cp %{SOURCE7} docs/.
+cp %{SOURCE8} docs/.
+cp %{SOURCE9} .
+cp %{SOURCE10} 000-shortnames.conf
+cp %{SOURCE11} .
+cp %{SOURCE12} policy.json
+cp %{SOURCE13} .
```

Comment:
source13 is `containers-storage.conf.5.md`, should this line be `cp ... docs/.` like all the other `.md` files?

I'm going to be submitting one-by-one review comments because this is a lot to digest. My apologies.

Reply:
Actually after some thought ... the whole "cp some files to docs" thing is really squicky. Impossible to maintain. Would it make sense to revert back to the old `ls *.5.md` form (line 117)?

Reply:
> source13 is `containers-storage.conf.5.md`, should this line be `cp ... docs/.` like all the other `.md` files?

ohh thanks. I'll check and update.

> I'm going to be submitting one-by-one review comments because this is a lot to digest. My apologies.

Cool with me :) . Appreciate the reviews.

Reply:
PTAL. I added comments labelling the files along with each `cp %{SOURCEN}` line. Hope that makes things more readable. Also, sent SOURCE13 to docs/.